packages/tracing/src/transaction.ts

```diff
@@ -246,10 +246,7 @@ export class Transaction extends SpanClass implements TransactionInterface {
     const { publicKey: public_key } = client.getDsn() || {};
 
     const rate = this.metadata && this.metadata.transactionSampling && this.metadata.transactionSampling.rate;
-    const sample_rate =
-      rate !== undefined
-        ? rate.toLocaleString('fullwide', { useGrouping: false, maximumFractionDigits: 16 })
-        : undefined;
+    const sample_rate = rate !== undefined ? JSON.stringify(rate) : undefined;
```

Comment:
Do we even need the extra `JSON.stringify` call? Won't just doing `sample_rate: rate` below be enough?

Reply:
I think it should still be a string (`rate.toString()`) but apart from that, I think you're right

Reply:
Good point. We can simplify it further to that but I think but I think we still have to convert it toString (if I understand [this test](https://github.com/getsentry/relay/blob/dbb9763b6bbded61e6f790fc6cff3cc522dfd8b6/relay-sampling/src/lib.rs#L2196) correctly).